tests/DataSet/ObjectDataSetTest.php

```diff
@@ -401,6 +402,20 @@ public function testCaching(array $objectDataSets, int $expectedRulesCallsCount,
         $this->assertSame($expectedDataCallsCount, ObjectWithCallsCount::$getDataCallsCount);
     }
 
+    public function testWithoutCache(): void
+    {
+        $object1 = new stdClass();
```

Comment:
Maybe use data provider?

Reply:
It's cache testing. This is should work without dataProvide in one test.